assets/js/editor-components/default-notice/index.tsx

```diff
@@ -112,6 +98,30 @@ export function DefaultNotice( { block }: { block: string } ) {
 		saveEntityRecord,
 		slug,
 	] );
+	const noticeContent = createInterpolateElement(
+		__(
+			'If you would like to use this block as your default <blockTypeLink/>, <updateLink>update your page settings</updateLink>.',
+			'woo-gutenberg-products-block'
+		),
+		{
+			blockTypeLink: (
+				<span>
+					{ block === 'checkout'
+						? __( 'checkout', 'woo-gutenberg-products-block' )
+						: __( 'cart', 'woo-gutenberg-products-block' ) }
+				</span>
+			),
+			updateLink: (
+				// eslint-disable-next-line jsx-a11y/anchor-is-valid
+				<a href="#" onClick={ updatePage }>
```

Comment:
Clicking this multiple times will trigger the call several times, but that doesn't seem to cause a problem in my testing.